schedule/index.html

```diff
@@ -82,3 +83,38 @@
         <p>Please note that while we hope to avoid moving sessions around, this schedule is tentative and subject to change if necessary.</p>
     </div>
 </div>
+-->
+<div class="row white-background">
+  <div class="col-xs-12 col-md-12">
+    <p>
+      ScaleConf runs over <strong>three days</strong>. We start with one day of
+      fireside chats focussing on <strong>the business of scale</strong>, while the
+      other two days are presentations from local and international experts on
+      the <strong>technologies, problstrongs and solutions</strong> that they have
+      encountered in building sites and services that can handle the rigours of
+      today's internet.
+    </p>
+
+    <p>
+      The Business of Scale day focuses on the issues leaders and their teams
+      face when <strong>growing a technology-driven organisation</strong>. You'll get
+      to hear from those who've faced the challenges of building engineering
+      teams, growing healthy customer acquisition strategies, maintaining
+      inspiring working environments, balancing customer demands and technical
+      debt, dealing with exit opportunities and more.
+    </p>
+
+    <p>
+      The second and third days follow our usual formula of <strong>12 x 45 minute
+      talks</strong> plus regular breaks to keep the legs from falling asleep and
+      awesome lunches to feed the mind and body.
+    </p>
+
+    <p>
+      There will be <strong>three social events</strong> starting with the
+      Wednesday evening with a sponsored drinks event in the city. Then similar
+      events on the Thursday and Friday nights. These are a chance to socialise
+      and network with the speakers, organisers, and fellow conference attendees.
```

Comment:
Actually, just:

Each evening we will have a <strong>social event</strong>, to give you a chance to socialise and network with speakers, organisers, and your fellow conference attendees.